osgp/platform/osgp-adapter-kafka-distributionautomation/src/main/java/org/opensmartgridplatform/adapter/kafka/da/application/mapping/GridMeasurementPublishedEventConverter.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2020 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.kafka.da.application.mapping;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+
+import org.opensmartgridplatform.adapter.kafka.da.avro.AccumulationKind;
+import org.opensmartgridplatform.adapter.kafka.da.avro.Analog;
+import org.opensmartgridplatform.adapter.kafka.da.avro.AnalogValue;
+import org.opensmartgridplatform.adapter.kafka.da.avro.GridMeasurementPublishedEvent;
+import org.opensmartgridplatform.adapter.kafka.da.avro.MeasuringPeriodKind;
+import org.opensmartgridplatform.adapter.kafka.da.avro.Name;
+import org.opensmartgridplatform.adapter.kafka.da.avro.PhaseCode;
+import org.opensmartgridplatform.adapter.kafka.da.avro.PowerSystemResource;
+import org.opensmartgridplatform.adapter.kafka.da.avro.UnitMultiplier;
+import org.opensmartgridplatform.adapter.kafka.da.avro.UnitSymbol;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ma.glasnost.orika.CustomConverter;
+import ma.glasnost.orika.MappingContext;
+import ma.glasnost.orika.metadata.Type;
+
+/**
+ * Class for mapping String containing a measurement or congestion
```

Comment:
Congestion would probably not be needed, as they, as far as I know, go in the opposite direction.

Reply:
done